test/ScanDependencies/can_import_with_map.swift

```diff
@@ -12,11 +12,11 @@
 // RUN: echo "}," >> %/t/inputs/map.json
 // RUN: echo "{" >> %/t/inputs/map.json
 // RUN: echo "\"moduleName\": \"Swift\"," >> %/t/inputs/map.json
-// RUN: echo "\"modulePath\": \"%stdlib_module\"," >> %/t/inputs/map.json
+// RUN: echo "\"modulePath\": \"%/stdlib_module\"," >> %/t/inputs/map.json
 // RUN: echo "}," >> %/t/inputs/map.json
 // RUN: echo "{" >> %/t/inputs/map.json
 // RUN: echo "\"moduleName\": \"SwiftOnoneSupport\"," >> %/t/inputs/map.json
-// RUN: echo "\"modulePath\": \"%ononesupport_module\"," >> %/t/inputs/map.json
+// RUN: echo "\"modulePath\": \"%/ononesupport_module\"," >> %/t/inputs/map.json
```

Comment:
This fix is missing the removal of the comma after the last list element that I had in the other PR.

Reply:
Ah, good point.  That is optional - I can add it or it can be a follow up.  The reason that it is not needed is because the list itself contains objects.  Objects *can* have trailing commas.